UPGRADE.md

```diff
@@ -17,6 +17,10 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 * [shopsys/microservice-product-search]

```

Comment:
in [CHANGELOG.md and UPGRADE.md](https://github.com/shopsys/shopsys/commit/3b166da22bc3cb8ea5fb25bdb330a6e3a2bdc4c3) there is new package `microservice-product-search`
will `microservice-product-search-export` entry be in these files?